e2e-test/e2etest.py

```diff
@@ -0,0 +1,58 @@
+from helper import *
+import os
+
+# Setup configuration
+# These values should be injected into the environment before setup
+azure_token = os.environ.get("AZURE_TOKEN")
+bee_name = os.environ.get("BEE_NAME")
+billing_project_name = os.environ.get("BILLING_PROJECT_NAME")
+#TODO: Take in arguments from the command line?
+cbas=False
+wds_upload=True
+cbas_submit_workflow=False
+test_cloning=False
+
+def run_workspace_app_test(cbas, wds_upload, cbas_submit_workflow, test_cloning):
+
+    setup(bee_name)
+    header = {"Authorization": "Bearer " + azure_token};
+
+    # create the workspace
+    workspace_id, workspace_name = create_workspace(cbas, billing_project_name, header)
+
+    # track to see when the workspace WDS is ready to upload data into them
+    # sleep for 1 minute to allow apps to start up
+    if wds_upload or cbas_submit_workflow:
+        logging.info("Sleeping for 60 seconds while apps starts up")
+        time.sleep(60)

```

Comment:
Since we know that it will take some amount of time for the apps to start up, shouldn't I at least start with some minimum?  I can put it down to 30 so it's equivalent to the polling loop going once